presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveClient.java

```diff
@@ -2535,6 +2538,124 @@ public void testCreateTableUnsupportedType()
         }
     }
 
+    @Test
+    public void testCreateEmptyBucketedTemporaryTable()
+            throws Exception
+    {
+        List<ColumnMetadata> columns = ImmutableList.<ColumnMetadata>builder()
+                .add(new ColumnMetadata("id", VARCHAR))
+                .add(new ColumnMetadata("value", VARCHAR))
+                .build();
+        int bucketCount = 4;
+        List<String> bucketingColumns = ImmutableList.of("id");
+        MaterializedResult inputRows = MaterializedResult.resultBuilder(SESSION, VARCHAR, VARCHAR)
+                .build();
+
+        testCreateTemporaryTable(columns, bucketCount, bucketingColumns, inputRows);
```

Comment:
similarly , in this test class it's usually named as `doCreateTable`, etc. Looks like you are following the convention in `HiveIntegrationSmokeTest`

Reply:
`doCreateTable` implies that it only creates a table and doesn't check anything. `testCreateTemporaryTable` creates a table, and does all the verifications.